### PR TITLE
Create mute.js

### PR DESCRIPTION
**Pull request by shah1012**

### CODE REVIEW

File: commands/mute.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      channel: 'guild',
+    });
+  }
+
+  async exec(msg, args) {
+    if (!args.member) return msg.reply('No member found');
+
+    if (!args.time) return msg.reply('Please Specify a time!');  
+
+    let mmber = args.member;
```
Comment (SovietKitsune) [inline commands/mute.js:29]: Change to member

File: commands/mute.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+    // let mainrole = msg.guild.roles.cache.find(role => role.name  === "ROLENAMEHERE")
+    let MuteRole = msg.guild.roles.cache.get("484337500333277214");  //replace with server role id
```
Comment (SovietKitsune) [inline commands/mute.js:32]: Get id from the toml config
```js
const { parse } = require('toml');
const { readFileSync } = require('fs');

let { muted }= parse(readFileSync('config.toml'))
```

Comment (SovietKitsune) [inline commands/mute.js:32]: Prettier was not ran as it would replace double quotes to single quotes

File: commands/mute.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      aliases: ['mute', 'getmuted'],
+      args: [
+        {
+          id: 'member',
+          type: 'member',
+        },
+        {
+          id: 'time',
+          type: 'string',
+        },
```
Comment (SovietKitsune) [inline commands/mute.js:16]: Should default to false to signify that the mute is not going to be automatically removed

File: commands/mute.js (hunk 2 above)
Comment (shah1012) [reply commands/mute.js:32]: for muted would i still do 

```member.role.add(muted.id)```

Comment (SovietKitsune) [reply commands/mute.js:32]: You would do
```js
member.role.add(muted)
```

File: commands/mute.js [hunk 4]
```diff
@@ -24,26 +29,25 @@ class MuteCommand extends Command {
...
-        mmber.roles.remove(MuteRole.id) //removes mute role
-        msg.channel.send(`@${mmber.user.tag} has been unmuted`);
+    member.roles.add(muted);
+    msg.channel.send(`@${member.user.tag} has been muted for ${ms(ms(time))}`);
 
+    setTimeout(function () {
+      //person.roles.add(mainrole.id)
+      member.roles.remove(muted); //removes mute role
+      msg.channel.send(`@${member.user.tag} has been unmuted`);
     }, ms(time));
```
Comment (SovietKitsune) [inline commands/mute.js]: You should check if time is false, if it is, don't specify a time in the reply and don't set a timeout.

File: commands/mute.js [hunk 5]
```diff
@@ -24,26 +29,25 @@ class MuteCommand extends Command {
   async exec(msg, args) {
     if (!args.member) return msg.reply('No member found');
 
-    if (!args.time) return msg.reply('Please Specify a time!');  
+    if (!args.time) return msg.reply('Please Specify a time!');
```
Comment (SovietKitsune) [inline commands/mute.js:32]: This should be removed as if there is no time specified, the mute is unlimited.

File: commands/mute.js [hunk 6]
```diff
@@ -24,26 +29,25 @@ class MuteCommand extends Command {
...
 
-    let mmber = args.member;
+    let member = args.member;
 
     // let mainrole = msg.guild.roles.cache.find(role => role.name  === "ROLENAMEHERE")
-    let MuteRole = msg.guild.roles.cache.get("484337500333277214");  //replace with server role id 
+    //replace with server role id
 
-    if(!MuteRole) return msg.reply("Cant find the Mute Role!");
+    if (!muted) return msg.reply('Cant find the Mute Role!');
```
Comment (SovietKitsune) [inline commands/mute.js:39]: Should be `can not` or `couldn't`

**Review by SovietKitsune — CHANGES_REQUESTED** [review]

**Pull request closed without merging**